flags.css

```diff
@@ -8,6 +8,10 @@
     background-color: red;
     height: 50%
 }
+.indonesiaWhite {
+    background-color: white;
+    height: 50%
```

Comment:
I would probably also put something in the name about the 50%, but it's not that big of a deal.  If I saw "indonesiaWhite" I would assume it only had to do with colors, really.